index.html

```diff
@@ -65,46 +65,58 @@ <h2>
       Introduction
     </h2>
     <p>
-      This document defines an API enabling a website to request presentation
-      of a [=digital credential=].
+      This document defines an API enabling a website to request presentation 
+      and issuance of a [=digital credential=].
     </p>
     <p>
-      The API design is agnostic to both credential presentation [=digital
-      credential/exchange protocols=] and credential formats. However, to
+      The API design is agnostic to both credential 
+      [=digital credential/presentation requests|presentation=] [=digital
+      credential/exchange protocols=], credential 
+      [=digital credential/issuance request|issuance=] [=digital
+      credential/issuance protocols|protocols=] and credential formats. However, to
       promote interoperability this document includes a
       [[[#protocol-registry]]].
     </p>
     <p>
       The API is designed to support the following goals:
     </p>
     <ul>
-      <li>Keep the act of requesting separate from the specific [=digital
-      credential/protocol=], thereby enabling the extensibility of the
-      [=digital credential/protocol=] and credential formats.
+      <li>Keep the acts of [=digital credential/presentation requests|requesting=] 
+      and [=digital credential/issuance requests|issuing=] separate from 
+      the specific [=digital credential/exchange protocol=] and 
```

Comment:
Maybe we should change "exchange protocol" to "presentment protocol?

Reply:
If `exchange protocol` is to be changed, I would strongly advise `presentation protocol` rather than `presentment protocol`.